app/models/authentication.rb

```diff
@@ -99,4 +110,18 @@ def event_prefix
     when "ExtManagementSystem" then "ems"
     end
   end
+
+  # will be removed once moving to yaml format, no need to review method
```

Comment:
We are currently using an inventory file to deploy the cluster, `ansible_format` prepares the authentication entry for this file, this function will be removed once we move to using ansible config (same for all the rest of the authentication types).
Once we will move to use config.yml (should happen before this pr get merged) we will only use the `ansible_config` function that will need to stay in the authentication base class or take it down to all the subclasses that uses it (AuthenticationLdap ...) but then it won't be DRY.